src/Uno.Extensions.Reactive.UI.Markup/Uno.Extensions.Reactive.WinUI.Markup.csproj

```diff
@@ -6,11 +6,12 @@
 		<!--Temporary disable missing XML doc until fixed in the whole package-->
 		<WarningsNotAsErrors>$(WarningsNotAsErrors);CS1591</WarningsNotAsErrors>
 		<DefineConstants>$(DefineConstants);WINUI</DefineConstants>
-		<WinAppSdkVersion>1.2.221109.1</WinAppSdkVersion>
+		<WinAppSdkVersion>1.3.230602002</WinAppSdkVersion>
```

Comment:
@Youssef1313 was this required? If not, this should be reverted to avoid pushing devs to latest version of winappsdk unnecessarily